src/nvim/api/window.c

```diff
@@ -492,6 +492,30 @@ Dictionary nvim_win_get_config(Window window, Error *err)
   return rv;
 }
 
+/// Hides the window (like |:hide| with a |window-ID|).
```

Comment:
A bit miss-leading. It still closes the window, but the _buffer_ becomes hidden even when 'hidden' not set.

A line or two below quickly summarizing the difference with `nvim_win_close` would be useful.